docs/service-catalog/docs/13-01-service-brokers.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Service Brokers
+type: Service Brokers
+---
+
+A Service Broker is a server compatible with the [Open Service Broker API](https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md) specification. Each Service Broker registered in Kyma presents the services it offers to the Service Catalog and manages their lifecycle.
+
+The Service Catalog lists all services that the Service Brokers offer. Use the Service Brokers to:
+* Provision and de-provision an instance of a service
+* Create and delete a service binding
+
+Create a service binding to link a service instance to an application. During this process, credentials are delivered in Secrets to provide you with the information necessary to connect to the instance of a service. The process of deleting a service binding is known as unbinding. Each of the Service Brokers available in Kyma performs these operations in a different way. See the documentation of a given Service Broker to learn how it operates.
```

Comment:
I would combine this paragraph and the one above to have something like this:
```
The Service Catalog lists all services that the Service Brokers offer. Use the Service Brokers to:
* Provision and de-provision an instance of a service.
* Create and delete a service binding. When binding your service to an application, use Secrets containing credentials necessary to connect to the service instance. 

Each of the Service Brokers available in Kyma performs these operations in a different way. See the documentation of a given Service Broker to learn how it operates.

```
1. The question is if we should have a dot at the end of each sentence in the list - for sure for the second one, so maybe the first too to keep it consistent?
2. Please check the second bullet point if the info about secrets is correct (I mean whether I put it correctly because I might be wrong :)